src/Control/Eff/Choose.hs

```diff
@@ -34,6 +39,17 @@ import Control.Monad
 -- any constraints.
 newtype Choose a = Choose [a]
 
+instance ( MonadBase m m
+         , Typeable m
```

Comment:
@greydot these `Typeable` constraints seem spurious. two minor things before i approve this PR:
- [ ] if `Typeable` is indeed spurious, could you remove it and the `Data.Typeable` imports?
- [ ] bump up the version to `2.5.1.0` since this is a non-breaking change

Reply:
You're right, the Typeable constraint was redundant.